src/Config_Command.php

```diff
@@ -223,14 +223,35 @@ public function get( $_, $assoc_args ) {
 
 		$wp_cli_original_defined_constants = get_defined_constants();
 		$wp_cli_original_defined_vars      = get_defined_vars();
+		$wp_cli_original_includes          = get_included_files();
 
 		eval( WP_CLI::get_runner()->get_wp_config_code() );
 
 		$wp_config_vars      = self::get_wp_config_vars( get_defined_vars(), $wp_cli_original_defined_vars, 'variable', array( 'wp_cli_original_defined_vars' ) );
 		$wp_config_constants = self::get_wp_config_vars( get_defined_constants(), $wp_cli_original_defined_constants, 'constant' );
 
-		$get_constant        = ! empty( $assoc_args['constant'] );
-		$get_global          = ! empty( $assoc_args['global'] );
+		foreach ( $wp_config_vars as $key => $value ) {
+			if ( 'wp_cli_original_includes' === $value['key'] ) {
+				$key_backup = $key;
+				break;
+			}
+		}
+
+		unset( $wp_config_vars[ $key_backup ] );
+		$wp_config_vars           = array_values( $wp_config_vars );
+		$wp_config_includes       = array_diff( get_included_files(), $wp_cli_original_includes );
+		$wp_config_includes_array = array();
+
+		foreach ( $wp_config_includes as $key => $value ) {
+			$wp_config_includes_array[] = array(
+				'key'   => 'FILE',
```

Comment:
> The case in the first colum are all CAPS except for table_prefix, so should I leave it as it is?

Oh, I see. In this case, instead of `FILE`, could you do `basename( $file_path )` so the filename appears?